ui/src/app/tasks/task-execution/task-execution.component.html

```diff
@@ -19,6 +19,12 @@
         <span class="fa fa-play"></span>
         Relaunch task
       </button>
+      <button name="task-launch" type="button" id="task-stop"
```

Comment:
This element moves down to the next line (because of the alignment on `app-page-head-actions`) and overlaps with the table layout below.

For the time being, I will fix the alignment by reducing the button name length when merging this PR.